warplib/ecrecover.cairo

```diff
@@ -26,6 +29,20 @@ func ecrecover_eth{range_check_ptr, bitwise_ptr : BitwiseBuiltin*, keccak_ptr :
 
     let v = v - 27
 
+    let (r_eq_0) = warp_eq256(_r, Uint256(0, 0))
+    let (s_eq_0) = warp_eq256(_s, Uint256(0, 0))
```

Comment:
This file is missing import for warp_eq256

Reply:
Fixed.